cli/src/pcluster/config/cluster_config.py

```diff
@@ -1373,12 +1401,19 @@ def _validate_mount_dirs(self):
             local_mount_dir_list=list(self.local_mount_dir_instance_types_dict.keys()),
         )
 
-    def _validate_new_storage_multiple_subnets(self, compute_subnet_ids, new_storage_count):
+    def _validate_new_storage_multiple_subnets(self, queues, compute_subnet_ids, new_storage_count):
         self._register_validator(
             ManagedFsxMultiAzValidator,
             compute_subnet_ids=compute_subnet_ids,
             new_storage_count=new_storage_count,
         )
+        for storage in self.shared_storage:
+            if isinstance(storage, (SharedFsxLustre, ExistingFsxOpenZfs, ExistingFsxOntap)) and storage.is_unmanaged:
+                self._register_validator(
+                    UnmanagedFsxMultiAzValidator,
+                    queues=queues,
+                    fsx_subnets=storage.file_system_availability_zones,
```

Comment:
wrong param name, subnet != az

Reply:
oops! ...good catch!
wonder why it was working...